### PR TITLE
Pollish up chinese translation (intro2jl)

### PR DESCRIPTION
**Pull request by inkydragon**
Based on #63 , may **merge it first**.

`intro-to-julia/`
- [x] 00.上手Jupyter notebook
- [x] 01.新手入门
- [x] 02.字符串
- [x] 03.数据结构
- [x] 04.循环
- [x] 05.条件判断
- [x] 06.函数
- [x] 07.包（Packages）
- [x] 08.绘图
- [x] 09.Julia很快
- [x] 10.多重派发
- [x] 10.1多重派发
- [x] 11.基本线性代数

all code test in julia 1.0.5 & sync with english version.

### CODE REVIEW

**Comment by NHDaly** [comment]
And same here:

LGTM, but we may want to invite review from a native-speaker. @Roger-luo?

File: .nbexports/zh-cn/intro-to-julia-ZH/01.新手入门.jl [hunk 1]
```diff
@@ -3,25 +3,26 @@
 # # 新手入门
 #
 # 主题：
-# 1. 如何打印（print）
-# 2. 如何给变量赋值
-# 3. 如何注释（comment）
-# 4. 基础数学语法
+# 1. 输出（print）
```
Comment (Roger-luo) [inline .nbexports/zh-cn/intro-to-julia-ZH/01.新手入门.jl:6]: 我觉得这里原来的翻译更好一些，字数减少以后感觉有点不清楚

File: .nbexports/zh-cn/intro-to-julia-ZH/03.数据结构.jl [hunk 2]
```diff
@@ -221,69 +232,76 @@ somemorenumbers[1] = 404
...
 @assert flexible_phonebook == Dict("Jenny" => 8675309, "Ghostbusters" => "555-2368")
 
 # ------------------------------------------------------------------------------------------
-# #### 3.4
+# ### 3.4
 # 给`flexible_phonebook`添加一个键为"Emergency"值为整数`911`的条目。
 # ------------------------------------------------------------------------------------------
 
-
+flexible_phonebook[""] = 911
```
Comment (Roger-luo) [inline .nbexports/zh-cn/intro-to-julia-ZH/03.数据结构.jl:291]: 为什么要加这一行？

Comment (inkydragon) [reply .nbexports/zh-cn/intro-to-julia-ZH/03.数据结构.jl:291]: 其实也可以不给的。
我看上面的题目风格类似代码改错，所以加了点题干，大概算提示？
题目比较简单，给不给题干都可以。

File: .nbexports/zh-cn/intro-to-julia-ZH/01.新手入门.jl (hunk 1 above)
Comment (inkydragon) [reply .nbexports/zh-cn/intro-to-julia-ZH/01.新手入门.jl]: done.

File: .nbexports/zh-cn/intro-to-julia-ZH/03.数据结构.jl (hunk 2 above)
Comment (Roger-luo) [reply .nbexports/zh-cn/intro-to-julia-ZH/03.数据结构.jl:291]: 嗯，我觉得尽量保持原来的代码比较好，这样中文文也方便对照。

Comment (inkydragon) [reply .nbexports/zh-cn/intro-to-julia-ZH/03.数据结构.jl]: 嗯，保持原状了

**Comment by logankilpatrick** [comment]
Hey @Roger-luo and thoughts on this? Should we merge?

**Comment by Roger-luo** [comment]
yeah looks good to me.

**Comment by logankilpatrick** [comment]
Great! Thank you all for the hard work on this and sorry for the delay on merging.

**Pull request merged**